src/styles/main.scss

```diff
@@ -110,7 +110,7 @@ label.v-label {
   }
 }
 
-@media only screen and (min-width: 1281px) and (max-width: 1600px) {
+@media only screen and (min-width: 1280px) and (max-width: 1600px) {
```

Comment:
Wollen wir in line 100 auch von 960 auf 961 hochsetzen?

Reply:
done